charts/rpe-feature-toggle-api/templates/dns.yaml

```diff
@@ -34,7 +34,7 @@ spec:
           command: [ "/bin/sh" ]
           args:
             - /container.init/feature-toggle-dns
-            - feature-toggle-api-{{ template "hmcts.releaseName" . }}
+            - feature-toggle-api-{{ .Release.Name }}
```

Comment:
@timja have tested this. issue is:

`consul reg=feature-toggle-api-probate-frontend-pr-568-ftr-tgl-api`
`host=feature-toggle-api-probate-frontend-pr-568.service.core-compute-preview.internal`